include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_coupled_solver.cpp

```diff
@@ -177,7 +177,7 @@ TimeIntBDFCoupled<dim, Number>::do_timestep_solve()
   }
 
   // explicit viscosity update
-  if(this->param.viscosity_is_variable() and
+  if(this->param.viscous_problem() and this->param.viscosity_is_variable() and
```

Comment:
:+1:

Reply:
I am starting to think we changed the bahevior of `viscosity_is_variable()` at some point and I completely overlooked that!